.github/workflows/wasm-release.yml

```diff
@@ -0,0 +1,61 @@
+name: WebAssembly Publish
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: src/api/js
+
+env:
+  EM_VERSION: 3.1.0
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Prepare for publish
+        run: |
+          npm version $(node -e 'console.log(fs.readFileSync("../../../scripts/release.yml", "utf8").match(/ReleaseVersion:\s*\x27(\S+)\x27/)[1])')
```

Comment:
I guess this is the version without "-pre" because I can't find a reference to "-pre" in the pull request.

Reply:
Yup, I deleted it just before opening this PR. Sorry that wasn't clearer.